jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTLSTest.java

```diff
@@ -834,6 +880,7 @@ protected boolean networkFlush(ByteBuffer output) throws IOException
 
         assertThrows(Exception.class, () -> client.newRequest("localhost", connector.getLocalPort()).scheme(HttpScheme.HTTPS.asString()).send());
 
+        byteBufferPool.asRetainableByteBufferPool().clear();
         await().atMost(5, TimeUnit.SECONDS).until(() -> leakedBuffers, is(empty()));
```

Comment:
I just noticed something unrelated to your change: the `leakedBuffers` variable should have a `CopyOnWriteArrayList`.